includes/validation/class-amp-validation-error-taxonomy.php

```diff
@@ -1824,11 +1824,14 @@ public static function filter_manage_custom_columns( $content, $column_name, $te
 								'edit.php'
 							)
 						),
-						esc_html( self::get_error_title_from_code( $validation_error['code'] ) )
+						esc_html( self::get_error_title_from_code( $validation_error ) )
 					);
 				}
 
-				if ( self::INVALID_ELEMENT_CODE === $validation_error['code'] || 'duplicate_element' === $validation_error['code'] ) {
+				$is_js_script = self::is_validation_error_for_js_script_element( $validation_error );
+				if ( self::INVALID_ELEMENT_CODE === $validation_error['code'] && $is_js_script && isset( $validation_error['node_attributes']['src'] ) ) {
+					$content .= sprintf( ': <code>%s</code>', esc_html( basename( wp_parse_url( $validation_error['node_attributes']['src'], PHP_URL_PATH ) ) ) );
+				} elseif ( ( self::INVALID_ELEMENT_CODE === $validation_error['code'] || 'duplicate_element' === $validation_error['code'] ) && ! $is_js_script ) {
```

Comment:
Eventually this whole conditional block should be moved to `\AMP_Validation_Error_Taxonomy::get_error_title_from_code()`, as opposed to checking conditions twice.

Reply:
Fixed in 96267a862ee13704122ef2f1014f91912ae38e13